frontend/src/components/Landing.vue

```diff
@@ -22,15 +23,20 @@ import TreeLogs from '../assets/images/tree-logs.jpg';
                     class="landing-button p-button-primary"
                     label="Login with IDIR"
                     @click="authService.methods.login"
-                    ><Icon icon="LoginIcon" small
-                /></Button>
+                >
+                    <Login />
+                </Button>
                 <Button
                     class="landing-button p-button-outlined"
                     label="Login with BCeID"
                     disabled
                     @click="authService.methods.login"
-                    ><Icon icon="LoginIcon" small
-                /></Button>
+                >
+                    <Icon
+                        icon="login"
+                        :size=Icon.medium
```

Comment:
As we discussed, this Icon.medium doesn't seems exists. Export the IconSize and use IconSize.